src/main/java/io/crate/jmx/recorder/NodeInfo.java

```diff
@@ -53,11 +54,15 @@ public boolean recordBean(String domain,
         } else if (attrName.equalsIgnoreCase("NodeName")) {
             LABEL_VALUES.put(NODE_NAME_LABEL, beanValue);
             validAttribute = true;
+        } else if (attrName.equalsIgnoreCase("ClusterStateVersion")) {
+            LABEL_VALUES.put(CLUSTER_STATE_VERSION_LABEL, beanValue);
+            validAttribute = true;
         }
-        if (LABEL_VALUES.size() == 2) {
-            List<String> labelValues = new ArrayList<>(2);
+        if (LABEL_VALUES.size() == 3) {
```

Comment:
Doesn't this break backward compatibility? (new exporter version won't function with older crate versions)

Is this tested somehow?

Reply:
Yes it will indeed. I don't know of any compatibility tests. I don't think we ever discussed what the policy should be (breaking changes should be released under a major release?)

Reply:
So far everything was afaik backward compatible. Couldn't we change the implementation here to keep this backward compatible?